guidelines/coding-guidelines.md

```diff
@@ -0,0 +1,6 @@
+## Performance related
+
+* Avoid allocations in hot paths:
```

Comment:
This has a colon at the end, but the following points are at the same level. Either:

- the colon should be removed, or
- this should not be a bullet, or
- the following bullets should be indented.